domino/routes.py

```diff
@@ -105,3 +99,8 @@ def deployment_version(self):
     # App URLs
     def app_publish(self):
         return self._build_project_url_private_api() + '/nb/startSession'
+
+    # Find Project By OwnerName and project name
+    def find_project_by_owner_name_and_project_name(self):
```

Comment:
The function name is misleading, this is just building the url as opposed to actually returning the actual value.